### PR TITLE
Add PyDataStructs to partners

### LINKED ISSUE #55: partner: Add new partner PyDataStructs
Add new partner to our webpage.

Partner: PyDataStructs
github: https://github.com/codezonediitj/pydatastructs
webpage: https://pydatastructs.readthedocs.io/en/stable/

### PR DESCRIPTION
**Pull request by luabida**
Fix #55

### CODE REVIEW

File: pages/partners.html [hunk 1]
```diff
@@ -77,4 +77,10 @@ <h5 class="card-title">pyOpenSci</h5>
       <a href="https://twitter.com/pyopensci"><i class="fa fa-brands fa-twitter-square"></i></p>
     </div>
   </div>
+  <div class="card">
+    <div class="card-body">
+      <a href=https://pydatastructs.readthedocs.io/en/stable style="text-decoration: None; "><h5 class="card-title">PyDataStructs</h5></a>
+      <p class="card-text">A Python package that contains common data structures.</p>
```
Comment (czgdp1807) [inline pages/partners.html]:
```suggestion
      <p class="card-text">A Python package for data structures and algorithms.</p>
```

**Review by czgdp1807 — CHANGES_REQUESTED** [review]

**Comment by czgdp1807** [comment]
Thanks for the addition @luabida. Requested a change in my review.

**Review by czgdp1807 — APPROVED** [review]

**Comment by luabida** [comment]
Thank you for the review! Branch is ready to merge.

**Review by xmnlab — APPROVED** [review]
LGTM! thanks @luabida for working on that.
thanks @czgdp1807 for the review :)

**Pull request merged**